src/components/edit/SizeDirective.js

```diff
@@ -0,0 +1,64 @@
+goog.provide('ga_size_directive');
+
+(function() {
+
+  var module = angular.module('ga_size_directive', []);
+
+  /**
+   * This directive add an interface where you can modify a glStyle.
+   */
+  module.directive('gaSize', function($window) {
```

Comment:
should be more explicit (also comment). this allows to change the width of e.g. streets. -> gaElementWidth